### ISSUE TITLE
Configure CodeBuild Setup - getting error

### ISSUE THREAD

**Issue by khalilj-appsflyer** [privacysandbox/aggregation-service#16]
When running the terraform code on step https://github.com/privacysandbox/aggregation-service/blob/main/build-scripts/aws/README.md#configure-codebuild-setup
I got the following error:

> │ Error: error creating S3 bucket ACL for aggregation-service-artifacts: AccessControlListNotSupported: The bucket does not allow ACLs

To resolve this error i had to add to: build-scripts/aws/terraform/codebuild.tf
The following resource:
```
resource "aws_s3_bucket_ownership_controls" "artifacts_output_ownership_controls" {
  bucket = aws_s3_bucket.artifacts_output.id

  rule {
    object_ownership = "BucketOwnerEnforced"
  }
}
```

**Comment by hostirosti** [comment]
Hi @khalilj-appsflyer,
thank you for reporting this issue. This is addressed with our latest release [0.10.0](https://github.com/privacysandbox/aggregation-service/releases/tag/v0.10.0), please let us know if you still see this error when deploying our latest release.

**Comment by hostirosti** [comment]
This has since been addressed. Closing this issue.
